src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">My name is {person.name}</h2>
+    <p className="Person__age">
+      {person.age !== undefined ? `I am ${person.age}` : ''}
+    </p>
+    <p className="Person__isMarried">
+      {person.isMarried === false ? (
+        'I am not married'
+      ) : (
+        <p className="Person__partner">
+          {person.partnerName !== undefined
+            ? `${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`
+            : ''}
+        </p>
+      )}
```

Comment:
It is bad practice to nest multiple ternary operators, move these checks to separate variables